discovery/bootstrapper.go

```diff
@@ -348,7 +353,9 @@ search:
 
 				// Finally we'll convert the host:port peer to
 				// a proper TCP address to use within the
-				// lnwire.NetAddress.
+				// lnwire.NetAddress. We don't need to use
+				// the lndResolveTCP function here because we
```

Comment:
It seems the `lndResolveTCP` was present in the past, but has been removed in the latest diff.